api/src/main/java/net/labymod/addons/waypoints/waypoint/Waypoint.java

```diff
@@ -19,4 +19,9 @@ default Color color() {
   default WaypointType type() {
     return this.meta().getType();
   }
+
+  @Override
+  default boolean isSeeThrough() {
+    return true;
```

Comment:
When enabling this, you should consider handling the "darker" parameter in the renderInWorld method (see DefaultWaypoint). This is to get a similar behavior for player name tags that are rendered through walls but slightly darker.

You can have a look at the MarkerObject class in LabyMod which basically just uses the following color for rendering (alpha 75 when darker is true):
```java
ColorUtil.toValue(255, 255, 255, darker ? 75 : 255)
```